README.md

```diff
@@ -1,5 +1,4 @@
-# curriculum-cr-training-mobile-menu
-
+# My Portfolio
 
 This repo host an initial code for the tasks performed during the Code Reviewers training.
 
```

Comment:
Kindly add a descriptive `README.md` file to the project. Adding a descriptive `README.md` file makes your project look professional and attractive to fellow programmers and recruiters since that is the first thing they would read when they want to take a look at your source code.

Reply:
## OPTIONAL
- [ ] Kindly when I would suggest adding a screenshot of your project in the README